rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -222,7 +226,9 @@ http {
         ''               close;
     }
 
-    map {{ buildForwardedFor $cfg.ForwardedForHeader }} $the_real_ip {
+    # The following is a sneaky way to do "set $the_real_ip $remote_addr"
```

Comment:
I'd rather use a Go template comment to avoid writing this message to the config file.

Reply:
IMO this could be useful to someone `cat`ing the rendered config from a running container, to help understand what's going on. 

The rendered template already contains other similar comments, and this one is rendered just once so it's not like it'd cause much bloat.

Reply:
You're right, no strong opinion on that one.